examples/plot_synthetic.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+============================================
+Synthetic dataset example with visualization
+============================================
+"""
+
+# %%
+from fairlearn.datasets._synthetic import (
+    SensitiveDatasetMaker,
+    SensitiveFeature,
+)
+from fairlearn.metrics import (
+    MetricFrame,
+    false_positive_rate,
+    true_positive_rate,
+    selection_rate,
+    count
+)
+import numpy as np
+from sklearn.metrics import accuracy_score, precision_score, recall_score
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeClassifier
+
+rng = np.random.RandomState(42)
+
+gender_feature = SensitiveFeature('Gender', ['Man', 'Other', 'Unspecified', 'Woman'])
+age_feature = SensitiveFeature('Age', ['<21', '21-64', '>64'])
+
+dataset = SensitiveDatasetMaker(sensitive_features=[gender_feature, age_feature], random_state=rng)
+
+# need a convenience method here
+dataset.configured_groups[('Man', '<21')].classification_kwargs['n_samples'] = 100
```

Comment:
don't we need to skew the parameter passed to `make_classification`? Otherwise the classes will be indistinguishable for the model.